### ISSUE TITLE
Checkbox doesn't Show Focus Indicator

### ISSUE THREAD

**Issue by Lijing29** [OfficeDev/office-ui-fabric-react#10320]
### Environment Information

- **Package version(s)**: v7
- **Browser and OS versions**: Chrome + Edge

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/lijinglin29/pen/VwZzwMX

#### Actual behavior:

The focus indicator is not shown when using tab to navigate the checkbox.

#### Expected behavior:

Show focus indicator

**Comment by maxwellred** [comment]
Thank you @Lijing29 I see the issue and will look into it shortly!

**Comment by maxwellred** [comment]
Actually I was able to focus and select (with spacebar) the checkboxes in both chrome and edge. Is there maybe something in the repro I missed?

![Screen Shot 2019-08-30 at 11 12 27 AM](https://user-images.githubusercontent.com/13246181/64042610-1647c780-cb17-11e9-9ede-3f1741dbf9f1.png)

![Screen Shot 2019-08-30 at 11 12 00 AM](https://user-images.githubusercontent.com/13246181/64042595-07611500-cb17-11e9-8362-76e4a18add1d.png)

**Comment by Lijing29** [comment]
Do you test it in codepen? The example given on the official website can show the focus indicator. However, the examples in codepen can't.
